mailbox/api/src/main/java/org/apache/james/mailbox/MailboxManager.java

```diff
@@ -80,7 +80,8 @@
         Annotation,
         Move,
         Namespace,
-        UserFlag
+        UserFlag,
+        RightSearch
```

Comment:
It's more **ACLList** because a mailbox manager could very well support ACL storage but not Listing mailboxes a user has right on. 

I know, sounds stupid... But you can still do valid things out of this.

I will however rename this...